bazel/repository_locations.bzl

```diff
@@ -101,12 +101,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "c-ares",
         project_desc = "C library for asynchronous DNS requests",
         project_url = "https://c-ares.haxx.se/",
-        version = "1.16.1",
-        sha256 = "d08312d0ecc3bd48eee0a4cc0d2137c9f194e0a28de2028928c0f6cae85f86ce",
+        version = "4d6975b3ec24aba44331d235bbcabf899be56cf2",
+        sha256 = "03f8470e6c6e39110557ff3d23d828c70267a1a057ef9b6e34215b6d4c7b8d6c",
         strip_prefix = "c-ares-{version}",
-        urls = ["https://github.com/c-ares/c-ares/releases/download/cares-{underscore_version}/c-ares-{version}.tar.gz"],
+        urls = ["https://github.com/c-ares/c-ares/archive/{version}.tar.gz"],
         use_category = ["dataplane_core", "controlplane"],
-        last_updated = "2020-05-11",
+        last_updated = "2020-10-19",
```

Comment:
We're switching to make this release (commit) date. I think this is 2020-09-28 based on https://github.com/c-ares/c-ares/commit/4d6975b3ec24aba44331d235bbcabf899be56cf2.

Reply:
Updated branch from master (update last_updated -> release_date https://github.com/envoyproxy/envoy/pull/13644#issuecomment-713103944), fixed the date.